package.json

```
@@ -15,24 +15,24 @@
     "url": "https://github.com/standard/standard/issues"
   },
   "dependencies": {
-    "eslint": "^8.13.0",
-    "eslint-config-standard": "17.0.0",
+    "eslint": "^8.41.0",
+    "eslint-config-standard": "17.1.0",
     "eslint-config-standard-jsx": "^11.0.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-n": "^15.5.1",
-    "eslint-plugin-promise": "^6.0.0",
-    "eslint-plugin-react": "^7.28.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^15.7.0",
```

Should we bump/allow 16.x here?

```suggestion
    "eslint-plugin-n": "^15.7.0 || ^16.0.0",
```

---

@LinusU We can't without bumping the required Node to 16, and that would require us to release as Standard 18